comprehensive-review/solutions/garden-streams/src/main/java/com/redhat/garden/sensors/SensorMeasurement.java

```diff
@@ -0,0 +1,20 @@
+package com.redhat.garden.sensors;
+
+import java.util.Date;
+
+public class SensorMeasurement {
```

Comment:
This measurement is an event? Because it is not named like an event. Perhaps we can name it something different like "SensorMeasurementCreated" (or similar). In my app I have one event named "SensorMeasurement" because is a SSE, not a real EDA event. For that reason if we use a similar name for the event that we persist in Kafka.. it might look weird in the REST app.

Reply:
Events are in the `event` folder. Only those are events